sdk/docs/pages/typescript-sdk/faucet.mdx

```diff
@@ -1,21 +1,23 @@
-import { Callout } from 'nextra/components';
-
 # Faucet
 
-To get started with the SDK it can be helpful get some Sui tokens to play with. The faucet is a
-simple way to get some tokens sent to your address in `testnet`, `devnet` or `localnet` if you are
-running your own network locally for development.
+import { Callout } from 'nextra/components';
+
+Devnet, Testnet, and local networks include faucets that mint SUI. You can use the Sui TypeScript SDK to call a network's faucet and provide SUI to the address you provide.
 
-You can request sui from the faucet using the `@mysten/sui.js/faucet` package.
+To request SUI from a faucet, import the `requestSuiFromFaucetV0` function from the `@mysten/sui.js/faucet` package to your project. 
 
 ```typescript
 import { requestSuiFromFaucetV0, getFaucetHost } from '@mysten/sui.js/faucet';
+```
+Use `requestSuiFromFaucetV0` in your TypeScript code to request SUI from the network's faucet. The `getFaucetHost` function is useful for development, but you should use the actual [Mainnet RPC address](../index#network-locations) as the `host` parameter value in production apps. 
```

Comment:
there won't be a faucet in mainnet, so this callout only really applies for setting up a SuiClient and not faucet methods

Reply:
Ha! Duh. I wasn't thinking...